docs/assets/css/style.scss

```diff
@@ -2,9 +2,42 @@
 ---
 
 @import "{{ site.theme }}";
+
+// Variables
+
+$font-heading: "Fredoka One", sans-serif;
+$font-body: "Lora", serif;
+
+// Style declarations
+
 .page-header {
   background-image: url(https://github.com/google/ground-platform/raw/master/docs/assets/images/aerial-forest-lake.jpg);
   background-repeat: no-repeat;
   background-position: center;
   background-size: cover;
 }
+
+h1, h2, h3, h4, h5, h6 {
+  font-family: $font-heading;
+  text-transform: uppercase;
+} 
+
+.main-content p {
+  font-family: $font-body;
+}
+
+.project-name {
+  font-family: $font-heading;
+  font-variant: all-small-caps;
+  font-weight: lighter;
+  letter-spacing: 1pt;
+  font-size: 4.5rem;
+}
+
+.project-tagline {
+  font-family: sans-serif;
+  font-weight: normal;
+  margin-bottom: 2rem;
+  font-variant: all-small-caps;
+  text-transform: none;
+}
```

Comment:
Also should we override opacity to 0.8 here? It was kinda hard to read on the current background.